feat(docs): improving docs for local install

djnnvx opened the pull request:
# Description

this should help people install the app for development and speed up new features & onboarding.

# Fixes
N/A

# Changes include
- specifying local installation recommending `asdf-vm` as a manager 
- providing links in every other readme to this file
- dropping function to install pip dependencies from `setup.sh`
- removing pointless docs from `api/` folder 

# Breaking changes
`./toolbox/scripts/setup.sh` does not include pip modules installation as it can be redundant & `scripts.sh` should only manage database initial setup & installation.

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added @vrn-sh/members to the reviewers, or specific members of the team
- [x] I have added the needed labels
- [x] I have tested this code
- [x] I have added / updated tests (unit / functionals / end-to-end / ...)
- [x] I have updated the README and other relevant documents (guides...)

# Additional comments

Feel free to add anything that could be relevant as a comment, id be happy to add it.

djnnvx committed:
feat(docs): improving docs for local install

this should help people install the app for development and speed up new
features & onboarding.

djnnvx commented:
@TheOrionsB is there anything particular to add to the front-end regarding the changes you made or is it all good ?

lambdina commented on README.md:
```diff
@@ -38,8 +38,7 @@ docker-compose up
-directories, as they will provide more informations regarding how to run the app locally.
+Please follow [this link](./toolbox/docs/INSTALL_LOCALLY.md).
 
 ## how to contribute ?
```
Maybe putting it directly in a CONTRIBUTE.md ??

djnnvx replied:
CONTRIBUTE is better suited for Pull Request etiquette no? This is not a bad idea, but maybe once the project gets bigger

djnnvx committed:
feat(docs): addressing PR comments+using mapped domain name

nginx can be capricious when not using a standardized domain name, so
that's the solution we will use from now on

djnnvx committed:
fix(ci): docker-compose logs should not follow logs
